.azdevops/CI.yml

```
@@ -27,51 +27,52 @@ stages:
     displayName: Build module
     steps:
 
+    - task: UseDotNet@2
+      displayName: 'Get .NET 7.0 SDK'
+      inputs:
+        packageType: sdk
+        version: 7.x
+        includePreviewVersions: true
+
     - pwsh: |
         & $(Build.SourcesDirectory)\SimpleBuild.ps1
```

The `&` shouldn't be needed here, although it might be better to wrap the whole path in double quotes in case there's whitespace then you would need the `&`

```suggestion
        & "$(Build.SourcesDirectory)\SimpleBuild.ps1"
```

---

Fixed.